features/mbedtls/targets/TARGET_NUVOTON/TARGET_M480/des/des_alt.h

```diff
@@ -13,16 +13,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
- 
+
 #ifndef MBEDTLS_DES_ALT_H
 #define MBEDTLS_DES_ALT_H
 
-#if !defined(MBEDTLS_CONFIG_FILE)
-#include "mbedtls/config.h"
-#else
-#include MBEDTLS_CONFIG_FILE
-#endif
```

Comment:
@andresag01 `mbed-os/features/mbedtls/inc/mbedtls/des.h` is included in front and it would handle configuration. This follows suggestion of @RonEld in https://github.com/ARMmbed/mbed-os/pull/4608.
> no real need, as aes.h includes config.h before including aes_alt.h , and aes_alt.h should not be included in any other location